airbyte-webapp/src/components/CreateConnection/TryAfterErrorBlock.tsx

```diff
@@ -16,8 +16,13 @@ export const TryAfterErrorBlock: React.FC<TryAfterErrorBlockProps> = ({ message,
   <div className={styles.container}>
     <StatusIcon big />
     <Text as="p" size="lg" centered className={styles.message}>
-      {message || <FormattedMessage id="form.schemaFailed" />}
+      <FormattedMessage id="form.schemaFailed" />
     </Text>
+    {message && (
+      <Text as="p" size="lg" centered className={styles.message}>
+        <FormattedMessage id="form.error" />: {message}
```

Comment:
Ideally, the format of this is something along the lines of:

```
FormattedMessage id="form.errorMessage" values={{ message }}
```

Where errorMessage is: `"Error: {message}"`